azure-kusto-ingest/tests/e2e.py

```diff
@@ -32,15 +36,15 @@
 class TestE2E(unittest.TestCase):
     """A class to define mappings to deft table."""
 
-    engine_cs = None  # type: Optional[str]
-    dm_cs = None  # type: Optional[str]
-    app_id = None  # type: Optional[str]
-    app_key = None  # type: Optional[str]
-    auth_id = None  # type: Optional[str]
-    test_db = None  # type: Optional[str]
-    client = None  # type: KustoClient
-    test_table = None  # type: str
-    current_count = None  # type: int
+    engine_cs: "Optional[str]" = None
```

Comment:
Why the quotation marks?

Reply:
Variable annotations at the class level are evaluated (they aren't for local variables), and since I only import Optional with TYPE_CHECKING, they need to be quoted.